File: src/main/java/com/mallang/auth/exception/UnsupportedOauthTypeException.java
package com.mallang.auth.exception;

import static org.springframework.http.HttpStatus.INTERNAL_SERVER_ERROR;

import com.mallang.common.execption.ErrorCode;
import com.mallang.common.execption.MallangLogException;

public class UnsupportedOauthTypeException extends MallangLogException {

    public UnsupportedOauthTypeException() {
        super(new ErrorCode(INTERNAL_SERVER_ERROR, "Oauth 에 문제가 있습니다."));
    }
}

File: src/main/kotlin/com/mallang/auth/exception/UnsupportedOauthTypeException.kt
package com.mallang.auth.exception

import com.mallang.common.execption.ErrorCode
import com.mallang.common.execption.MallangLogException
import org.springframework.http.HttpStatus.INTERNAL_SERVER_ERROR

class UnsupportedOauthTypeException : MallangLogException(
        ErrorCode(INTERNAL_SERVER_ERROR, "Oauth 에 문제가 있습니다.")
)
